study-builder/config/circadia-config.json.ctmpl

```diff
@@ -0,0 +1,38 @@
+{{$environment := env "ENVIRONMENT"}}
+{{$version := env "VERSION"}}
+{{with $response := vault (printf "secret/pepper/%s/%s/circadia/conf" $environment $version)}}
+{{with $conf := $response.Data}}
+
+{
+  "AUTH0_DOMAIN": "{{$conf.auth0.deployDomain}}",
+  "AUTH0_CLIENT_ID": "{{$conf.auth0.deployClientId}}",
+  "AUTH0_CLIENT_SECRET": "{{$conf.auth0.deployClientSecret}}",
+  "AUTH0_KEYWORD_REPLACE_MAPPINGS": {
+  {{if eq $environment "dev"}}
+    "LOCALHOST_CALLBACKS": ",http://localhost:4200/auth,http://localhost:4200/login-landing,https://circadia-dot-broad-ddp-dev.uc.r.appspot.com/auth,https://circadia-dot-broad-ddp-dev.uc.r.appspot.com/login-landing",
+    "LOCALHOST_LOGOUTS": ",http://localhost:4200,http://localhost:4200/error,http://localhost:4200/login-redirect,http://localhost:4200/session-expired,https://circadia-dot-broad-ddp-dev.uc.r.appspot.com,https://circadia-dot-broad-ddp-dev.uc.r.appspot.com/error,https://circadia-dot-broad-ddp-dev.uc.r.appspot.com/login-redirect,https://circadia-dot-broad-ddp-dev.uc.r.appspot.com/session-expired",
+    "LOCALHOST_ORIGINS": ",http://localhost:4200,https://circadia-dot-broad-ddp-dev.uc.r.appspot.com",
+  {{else}}
+    "LOCALHOST_CALLBACKS": "",
+    "LOCALHOST_LOGOUTS": "",
+    "LOCALHOST_ORIGINS": "",
+  {{end}}
+    "BASE_URL": "{{$conf.baseWebUrl}}",
+    "LOGIN_DOMAIN": "{{$conf.auth0.loginDomain}}",
+    "ASSETS_URL": "{{$conf.assetsBucketName}}",
```

Comment:
This one is a bit confusing -- we have a bucket name but we're saying it's a URL?